gci/views.py

```diff
@@ -16,6 +17,17 @@ def index(request):
     org_id = linked_students[0]['organization_id']
     org_name = linked_students[0]['organization_name']
     s = []
+
+    favicon = get_logo(org_name, 16)
+    with open('favicon.png', 'wb') as favicon_file:
+        favicon_file.write(favicon)
+
+    org_logo = get_logo(org_name)
+    with open('org_logo.png', 'wb') as org_logo_file:
+        org_logo_file.write(org_logo)
```

Comment:
The build fails here with:

`CommandError: Failed to render view: a bytes-like object is required, not 'Response'`

because you made get_logo return the requests Response object, not a blob.  Please fix `get_logo`